Library/Homebrew/manpages/brew.1.md

```diff
@@ -298,6 +298,10 @@ Note that these flags should only appear after a command.
 
     If no <formulae> are given, check all installed brews.
 
+  * `migrate` [<formulae>]:
```

Comment:
And it also can help if migrations of renamed formula fail on `update`. If they fail we might want to migrate packages manually.